testsuite/MDAnalysisTests/coordinates/test_timestep_api.py

```diff
@@ -162,17 +163,17 @@ def test_coordinate_setter_shortcuts(self, ts):
     # n_atoms should be a read only property
     # all Timesteps require this attribute
     def test_n_atoms(self, ts):
-        assert_equal(ts.n_atoms, ts._n_atoms)
+        assert_equal(ts.n_atoms, self.size)
 
     def test_n_atoms_readonly(self, ts):
         with pytest.raises(AttributeError):
             ts.__setattr__('n_atoms', 20)
 
     def test_n_atoms_presence(self, ts):
```

Comment:
These `hasattr` tests are now basically useless as the attribute will **always** be there, but we can leave them in for posterity?